test/unit/report_importer_test.rb

```diff
@@ -10,7 +10,7 @@ class ReportImporterTest < ActiveSupport::TestCase
   end
 
   test 'it should import reports with no metrics' do
-    r = ReportImporter.import(read_json_fixture('report-empty.json'))
+    r = ConfigReportImporter.import(read_json_fixture('report-empty.json'))
```

Comment:
`report_importer_test` has been restored completely (https://github.com/theforeman/foreman/pull/2750/files#diff-0e5400bfc3caabc55745ba764d2b9ecd) besides the puppet error state tests where I have changed to `ConfigReportImporter`